zvmsdk_sles.spec

```diff
@@ -12,7 +12,7 @@ License: ASL 2.0
 BuildArch: noarch
 Group: System/tools
 Autoreq: no
-Requires: python >= 2.7, python-netaddr >= 0.7.5, python-PyJWT >= 1.0.1, python-requests >= 2.6.0, python-Routes >= 2.2, python-WebOb >= 1.2.3, python-jsonschema >= 2.3.0, python-six >= 1.9.0, zthin >= 3.1.0
+Requires: python3 >= 2.7, python3-netaddr >= 0.7.5, python3-WebOb >= 1.7.4, python3-PyJWT >= 2.4.0, python3-pyroute2, python3-requests >= 2.6.0, python3-jsonschema >= 2.3.0, python3-six >= 1.9.0, zthin >= 3.1.0, apache2 >= 2.4.0, apache2-mod_wsgi > 4.7.0
```

Comment:
This list includes  `python3-pyroute2`.

I thought we had a problem with that one, and that we needed to use pip?

Please clarify, I am lost.